Dockerfile.fluentbit

```diff
@@ -1,5 +1,5 @@
-ARG VERSION
 FROM registry.access.redhat.com/rhel7:latest
+ARG VERSION
```

Comment:
Do you have tooling somewhere that required this change?  Just trying to learn.  AFAICS it shouldn't matter either way: from https://docs.docker.com/engine/reference/builder/#format: "A Dockerfile must begin with a `FROM` instruction. This may be after parser directives, comments, and globally scoped ARGs."

Reply:
same page https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact ,
"An ARG declared before a FROM is outside of a build stage, so it can’t be used in any instruction after a FROM."
It wouldn't build without it for me.

Reply:
TIL! thanks!